BuildAndTest.proj

```diff
@@ -48,7 +43,7 @@
       <TestAssemblies Include="Binaries\$(Configuration)\**\$(IncludePattern)" />
     </ItemGroup>
 
-    <Exec Command="Binaries\$(Configuration)\RunTests.exe packages\xunit.runners.2.0.0-alpha-build2576\tools $(RunTestArgs) @(TestAssemblies, ' ')" />
+    <Exec Command="Binaries\$(Configuration)\RunTests.exe packages\xunit.runner.console.2.1.0-beta4-build3109\tools $(RunTestArgs) @(TestAssemblies, ' ')" />
```

Comment:
Any chance we can avoid the version hardcode? We've been bitten a lot by NuGet versions being baked in many places and not updating all of them.